app/controllers/api/v1/verify_account_controller.rb

```diff
@@ -51,6 +44,17 @@ def activate
 
         render_data status: :ok
       end
+
+      private
+
+      def pre_token_creation
```

Comment:
I would rename this to something more descriptive - find_user_and_authorize or something like that